apps/emqx_gateway_jt808/src/emqx_jt808_schema.erl

```diff
@@ -0,0 +1,118 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_jt808_schema).
+
+-include("emqx_jt808.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+-include_lib("typerefl/include/types.hrl").
+
+-export([fields/1, desc/1]).
+
+-define(NOT_EMPTY(MSG), emqx_resource_validator:not_empty(MSG)).
+
+fields(jt808) ->
+    [
+        {frame, sc(ref(jt808_frame))},
```

Comment:
Miss the `mountpoint` option

Reply:
should the mountpoint be support to all gateway app?

Reply:
Mountpoint tested. 🚀